examples/pipelines/datacomp/pipeline.py

```diff
@@ -41,64 +41,66 @@
         "dataset_name": "nielsr/datacomp-small-with-text-embeddings",
         "column_name_mapping": load_component_column_mapping,
         "index_column": "uid",
-        # "n_rows_to_load": 1000,
+        "n_rows_to_load": 1000,
+    },
+    node_pool_label="node_pool",
+    node_pool_name="n2-standard-64-pool",
+    cache=False,
+)
+download_images_op = ComponentOp.from_registry(
+    name="download_images",
+    arguments={
+        "retries": 2,
+        "min_image_size": 0,
+        "max_aspect_ratio": float("inf"),
+    },
+    node_pool_label="node_pool",
+    node_pool_name="n2-standard-64-pool",
+    input_partition_rows=1000,
+    cache=False,
+)
+detect_text_op = ComponentOp(
+    component_dir="components/detect_text",
+    arguments={
+        "batch_size": 2,
+    },
+    node_pool_label="node_pool",
+    node_pool_name="model-inference-mega-pool",
+    number_of_accelerators=1,
+    accelerator_name="GPU",
```

Comment:
Not something we need to tackle in this PR, but we might want to make the infrastructure arguments dynamic instead by eg. accepting a dict that we pass on.